### ISSUE TITLE
Bug in AlignIO.read function renders phylip-sequential output unusable

### ISSUE THREAD

**Issue by matthew-fogel** [biopython/biopython#1552]
### Setup

I am reporting a problem with Biopython version, Python version, and operating
system as follows:

```
>>> import sys; print(sys.version)
3.5.2 (default, Nov 23 2017, 16:37:01) 
[GCC 5.4.0 20160609]
>>> import platform; print(platform.python_implementation()); print(platform.platform())
CPython
Linux-4.4.0-116-generic-x86_64-with-Ubuntu-16.04-xenial
>>> import Bio; print(Bio.__version__)
1.70
```

### Expected behaviour

When running AlignIO.write(alignments, output_location, "phylip-sequential") and the sequence names are ten characters or longer, they should be truncated with a space placed between the sequence name and the actual sequences.

### Actual behaviour

When the sequence names are ten characters or longer they are truncated to ten characters but no space is placed between the sequence name and the actual sequences.

### Steps to reproduce

Create a alignment file with sequence names ten characters or longer (e.g. read in a fasta file using AlignIo.read) then try to AlignIO.write(data_read_in, output_location, "phylip-sequential"). Open up the output file and you will see there is no space between sequences and sequence names.

**Comment by peterjc** [comment]
From memory, this is as per the original PHYLIP examples (which really did have no space). I would expect Biopython to be able to read the file in again file. If not, can you provide a self contained example?

**Comment by peterjc** [comment]
See the examples in the PHYLIP documentation linked to from our source code,
* http://evolution.genetics.washington.edu/phylip/doc/sequence.html
* http://evolution.genetics.washington.edu/phylip/doc/main.html#inputfiles

**Comment by matthew-fogel** [comment]
Thank you for you reply! The confusion was that a tool that I am using (Fasttree version 2.1.10) does not accomodate the PHYLIP specification and always assumes that there is a space between the sequence and sequence names in a PHYLIP format file. I will mark this issue as resolved.

**Comment by peterjc** [comment]
Biopython does have ``phylip-relaxed`` allowing long names which might work for you?

**Comment by matthew-fogel** [comment]
Thanks again! We found that when the input fasta sequences were exactly 50 characters in length and I call AlignIO.write(alignment, output_location, "phylip-relaxed") it outputs a number of blank lines after each PHYLIP record. Since I am boostrapping my FASTA data before writing it to an output file in PHYLIP format, I end up having multiple empty lines between each record. However I was able to access the record ids and sequences directly and this works for me.
